clang/test/dpct/nvshmem.cu

```diff
@@ -0,0 +1,148 @@
+// RUN: dpct --format-range=none -out-root %T/nvshmem %s --cuda-include-path="%cuda-path/include" --extra-arg="-I%cuda-path/../../nvshmem/latest/include"
+// RUN: FileCheck %s --match-full-lines --input-file %T/nvshmem/nvshmem.dp.cpp
```

Comment:
could you enable the build test for the lit?   (just make sure the migrated API can be build). 

Ref to: wmma.cu

Reply:
The test passes locally

![image](https://github.com/user-attachments/assets/670758f9-cbf9-45c6-82ba-3ce045ea0cc0)